packages/excalidraw/element/bindingTest.ts

```diff
@@ -0,0 +1,233 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Don't disable linting and formatting. Please set up your development environment to follow linter rules (preferably auto-format), so we can easily review your code.

Reply:
I am working on it 👍